app/frontend/src/store/modules/user.js

```diff
@@ -37,6 +53,21 @@ export default {
             console.error(error); // eslint-disable-line no-console
           });
       }
+    },
+    /**
+     * @function fillInAcronymClientStatus
+     * Fetch the service clients for the acronyms the user has
+     * @param {object} context The store context
+     */
+    async fillInAcronymClientStatus(context) {
+      try {
+        const response = await UserService.getServiceClients(
+          Vue.prototype.$keycloak.subject
```

Comment:
This may not be safe to call yet without the prerequisite keycloak ready and authenticated checks. Will have to investigate with the authentication abstraction refactor.

Reply:
Indeed, didn't want to do that multi-property check. We'll refactor it a bit after discussion tomorrow.

Reply:
Will leave this and revisit with Jeremy after auth state changes.

Reply:
Will be using vuex lazy loading on view level to address on-demand loading.